data/monster/Bosses/the_horned_fox.xml

```diff
@@ -39,7 +39,7 @@
 	<summons maxSummons="6">
 		<summon name="minotaur archer" interval="1000" chance="13" max="2" />
 		<summon name="minotaur guard" interval="1000" chance="13" max="2" />
```

Comment:
Looks like there are more instances of "max" in summon tags, any reason why you didn't remove all of them?